lib/checks/report/report.rb

```diff
@@ -0,0 +1,28 @@
+module Checks
+  module Report
+    class Report
```

Comment:
I assume you referring to the following line here? https://github.com/alphagov/finding-things-migration-checker/pull/18/files#diff-e496d5ba7c98bf92be59d9f4819dddbbR37

I don't know of a better way, but what if you added an `unless` statement, `require file unless file == 'report'` or something along those lines?